### PR TITLE
add workspace root to root label

### PR DESCRIPTION
**Pull request by marcellourbani**
I found it confusing to have several roots with the same name:

image

Like it much better with the workspace root added:
image
Best regards
Marcello

### CODE REVIEW

**Comment by rossknudsen** [comment]
See my comments on #28.

**Comment by marcellourbani** [comment]
replied to that

**Comment by marcellourbani** [comment]
image

old label was the type

File: src/ProjectManager.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ class ProjectManager {
 
     this.workspaceTestState = {
       id: "root",
-      label: this.repoParser.type,
+      label: `${this.workspace.name} (${this.repoParser.type})`,
```
Comment (rossknudsen) [inline src/ProjectManager.ts:69]: Cool, I understand what is happening here.

What do you think about dropping the ` (${this.repoParser.type})`?  I never intended for that value to be displayed, it was more for debugging purposes.  I don't think it is offering anything meaningful. If people wanted to know what RepoParser had been used, we could log it out.

Comment (rossknudsen) [inline src/ProjectManager.ts:69]:
```suggestion
      label: `${this.workspace.name}`,
```

Comment (marcellourbani) [reply src/ProjectManager.ts:69]: I don't care about that, I left it there because I thought you (or the original author) cared for it

**Pull request merged**